src/templates/main/404.html

```diff
@@ -1,5 +1,61 @@
-{% extends 'main/base.html' %}
```

Comment:
The same for template 500.html applies here. [Reference](https://github.com/Mosqlimate-project/Data-platform/pull/91#pullrequestreview-1659962261)